core/gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -71,6 +71,7 @@ export class WorkflowActionService {
   private tempWorkflow?: Workflow;
   private workflowModificationEnabled = true;
   private enableModificationStream = new BehaviorSubject<boolean>(true);
+  private enableHighlight = true;
```

Comment:
also, I suggest by default make it false. we turn it on explicitly in the workspace.